src/site/content/en/blog/speed-at-scale/index.md

```diff
@@ -0,0 +1,85 @@
+
+Until now, you've needed to solve lazy-loading images using a JavaScript library but that may soon change. This summer, Chrome will be launching support for the [loading](https://addyosmani.com/blog/lazy-loading/) attribute which brings native `<img>` and `<iframe>` lazy-loading to the web.
+
+<figure class="w-figure">
+  <img src="./Speed-at1.png" alt="Native lazy-loading highlighting offscreen content being loaded on-demand" loading="lazy">
+</figure>
+
+The `loading` attribute allows a browser to defer loading offscreen images and iframes until users scroll near them. `loading` supports three values:
+
+* `lazy`: is a good candidate for lazy loading.
+* `eager`: is not a good candidate for lazy loading. Load right away.
+* `auto`: browser will determine whether or not to lazily load.
+
+```html
+<img src="io2019.jpg" loading="lazy" alt="..." />
+<iframe src="video-player.html" loading="lazy"></iframe>
+```
+
+The exact heuristics for "when the user scrolls near" is left up to the browser. In general, our hope is that browsers will start fetching deferred images and iframe content a little before it comes into the viewport. 
+
+The `loading` attribute is implemented behind flags in Chrome Canary. You can try out [this demo](https://mathiasbynens.be/demo/img-loading-lazy) in Chrome 75+ with the `chrome://flags/#enable-lazy-image-loading` and `chrome://flags/#enable-lazy-frame-loading` flags turned on. 
+
+A [write-up](https://addyosmani.com/blog/lazy-loading/) on the native lazy-loading feature is available with more details.
+
+## Google fonts now supports font-display as a query parameter
+
+We announced support for [font-display](https://font-display.glitch.me) is now available in production for all [Google Fonts](https://fonts.google.com) via the [display](https://developers.google.com/fonts/docs/getting_started#use_font-display) query-string parameter:
+
+```html
+https://fonts.googleapis.com/css?family=Lobster&display=swap
+``` 
+
+The `font-display` descriptor lets you decide how your web fonts will render or fallback, depending on how long it takes for them to load. It supports a number of values including `auto`, `block`, `swap`, `fallback` and `optional`.
+
+Previously, the only way to specify font-display for web fonts from Google Fonts was to self-host them but this change removes the need to do so. 
+
+The [Google Fonts documentation](https://developers.google.com/fonts/docs/getting_started#use_font-display) has been updated to include `font-display` in the default code embeds (as seem below). We hope this will encourage more developers to try out this exciting addition.
+
+<figure class="w-figure">
+  <img src="./Speed-at2.png" alt="Google Fonts embed code with font-display included in the URL as a query-parameter"  loading="lazy">
+</figure>
+
+Here's a [demo](https://codepen.io/addyosmani/pen/RmaOVJ) on Codepen of using display with multiple font families.
```

Comment:
Could we turn this into an embedded Glitch?

Reply:
Happy to move this to Glitch. Will update the link in the next set of commits. Would prefer (for simplicity) to not make it embedded :)